tools/generator/src/Generator/SetTargetConfigurations.swift

```diff
@@ -357,6 +357,13 @@ $(CONFIGURATION_BUILD_DIR)
                 to: ldRunpathSearchPaths
             )
         }
+        
+        if target.product.type == .bundle {
+            buildSettings.set(
+                "CODE_SIGNING_ALLOWED",
+                to: false
+            )
```

Comment:
You only need to use `.set` if the type can't be inferred. This will work:
```suggestion
            buildSettings["CODE_SIGNING_ALLOWED"] = false
```